functionary/core/models/task.py

```diff
@@ -53,11 +63,15 @@ class Task(ModelSaveHookMixin, models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
     scheduled_task = models.ForeignKey(
-        ScheduledTask, null=True, blank=True, on_delete=models.SET_NULL
+        to="ScheduledTask", null=True, blank=True, on_delete=models.SET_NULL
```

Comment:
No, not in this case since this isn't for type checking in this context.  Django lets you pass in a string rather than the actual model class to avoid needing to do the import.  That's what's going on here.